src/icetransport.cpp

```diff
@@ -128,6 +129,44 @@ IceTransport::IceTransport(const Configuration &config, Description::Role role,
 
 	nice_agent_attach_recv(mNiceAgent.get(), mStreamId, 1, g_main_loop_get_context(mMainLoop.get()),
 	                       RecvCallback, this);
+
+	// Add TURN Servers
+	for (auto &server : servers) {
+		if (server.hostname.empty())
+			continue;
+		if (server.type == IceServer::Type::STUN)
+			continue;
+		if (server.service.empty())
+			server.service = "3478"; // TURN UDP port
+
+		struct addrinfo hints = {};
+		hints.ai_family = AF_INET; // IPv4
```

Comment:
You should set the family to `AF_UNSPEC` to also handle IPv6.

Reply:
OK

Reply:
Actually not OK.
We are resolving IP address of the server right? Why do we need IPV6 address?

Reply:
I mean that if the server has an IPv6 you should also add it to the relays. However this can be done in a second time.